darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -115,6 +123,20 @@ def fit(
         past_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
         future_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
     ):
+        """
+        Fits the forecasting models with the entire series except the last `regression_train_n_points` values.
```

Comment:
this needs to be extended with the regression (Ensemble) model being training

Reply:
Not sure of what you meant, let me know if the changes I applied correspond to your expectations.

Reply:
👍